packages/shared/src/components/buttons/ClickableText.tsx

```diff
@@ -11,29 +11,28 @@ export interface BaseClickableTextProps {
 }
 
 export type ClickableTextProps<Tag extends AvailableTags> =
-  BaseClickableTextProps &
-    JSX.IntrinsicElements[Tag] & {
-      innerRef?: LegacyRef<JSX.IntrinsicElements[Tag]>;
-    };
+  BaseClickableTextProps & JSX.IntrinsicElements[Tag];
 
 // eslint-disable-next-line @typescript-eslint/no-redeclare
-export function ClickableText<Tag extends AvailableTags>({
-  disabled,
-  pressed,
-  children,
-  tag: Tag = 'button',
-  innerRef,
-  defaultTypo = true,
-  className,
-  ...props
-}: ClickableTextProps<Tag>): ReactElement {
+export default function ClickableTextComponent<Tag extends AvailableTags>(
```

Comment:
Why do we export both components? I don't think we need the non forwardRef version.
This is very confusing